microprofile/config-smallrye/src/test/java/org/wildfly/extension/microprofile/config/smallrye/MPConfigSubsystemParsingTestCase.java

```diff
@@ -96,15 +102,52 @@ private void testRejectingTransformers(ModelTestControllerVersion controllerVers
         assertTrue(mainServices.isSuccessfulBoot());
         assertTrue(mainServices.getLegacyServices(microprofileConfigVersion).isSuccessfulBoot());
 
-        List<ModelNode> ops = builder.parseXmlResource("subsystem_1_0_reject_transformers.xml");
+        List<ModelNode> ops = builder.parseXmlResource("subsystem_reject_transformers.xml");
+        if (controllerVersion == ModelTestControllerVersion.EAP_7_3_0) {
+            // For 7.3.0 the ConfigSource 'add' operation installs services despite being in admin-only mode,
+            // which causes errors.
+            // Since the transformers are chained the rejection of the 'root' nested attribute installed
+            // by the 'my-config-source-with-root' config source is tested in later legacy controller versions.
+            int remove = -1;
+            for (int i = 0 ; i < ops.size() ; i++) {
+                if (ops.get(i).get("address").asString().contains("my-config-source-with-root")) {
+                    remove = i;
+                }
+            }
+            ops.remove(remove);
+        }
```

Comment:
No need to change this now, but if you ever want to clean this out go ahead as there is no longer a requirement to test transformation to prior to WF 23 / EAP 7.4.